games/static/games/proto.js

```diff
@@ -4,31 +4,56 @@ class Game {
     this.setup = setup;
     this.grid = document.querySelector(".grid");
     this.startButton = document.querySelector("#start");
+    this.restartButton = document.querySelector("#restart");
     this.isRunning = false
   }
   start() {
-    let alienMovementId = 0;
+    let gameId = 0;
+    let cleanId = 0;
+    let start
+    let restart
     let board = new Board(this.grid, BOARD_WIDTH, this.setup);
-    this.startButton.addEventListener("mousedown", () => {
-      if (alienMovementId) {
-        clearInterval(alienMovementId);
-        alienMovementId = null;
-        this.isRunning = false
+    this.startButton.addEventListener("mousedown", start = () => {
```

Comment:
is this working?
looks pretty weird...

Reply:
it looks like start & restart are fucntions
you can define them before the addEventListener. like:
```js 
let start = () => {...}
this.startButton.addEventListener("mousedown", start)
```